lineax/_misc.py

```diff
@@ -90,11 +90,11 @@ def __call__(self, *args, **kwargs):
         return self.fn(*args, **kwargs), None
 
 
-def jacobian(fn, in_size, out_size, has_aux=False):
+def jacobian(fn, in_size, out_size, holomorphic=False, has_aux=False):
```

Comment:
This needs piping into the `jacrev` branch as well.

Reply:
Looks like this branch is untested. Worth adding testcase somewhere